src/components/app-registration/AppRegistration/AppRegistrationForm/AppRegistrationForm.module.scss

```diff
@@ -4,8 +4,8 @@ fieldset .customTextInput:last-child {
 
 .customTextInput {
     align-items: center;
-    border: 1px solid var(--border-normal);
-    border-radius: 4px;
+    border: 0.1rem solid var(--border-normal);
+    border-radius: 0.4rem;
```

Comment:
Help to revert all the border changes, logically, font-size changes should not scale border/border-radius sizes.